gulpfile.babel.js

```diff
@@ -29,7 +29,7 @@ function loadConfig() {
 
 // Build the "dist" folder by running all of the below tasks
 gulp.task('build',
- gulp.series(clean, gulp.parallel(pages, sass, javascript, images, copy), styleGuide));
+ gulp.series(clean, gulp.parallel(pages, sass, fonts, javascript, images, copy), styleGuide));
```

Comment:
> I'm not totally sure where in the pipeline is best for font fetching via gulp

Since this is an asset, you can put it between `javascript` and `images`, or after `images` ;)